flintrock/flintrock.py

```diff
@@ -642,9 +626,9 @@ def stop(cli_context, cluster_name, ec2_region, ec2_vpc_id, assume_yes):
 @click.option('--ec2-spot-price', type=float)
 @click.option('--assume-yes/--no-assume-yes', default=False)
 @click.option('--ec2-tag', 'ec2_tags',
+              callback=ec2.validate_tags,
               multiple=True,
-              help="Additional tags (Key,Value pairs) to assign to the instances. "
-                   "You can specify this option multiple times.")
```

Comment:
Same here. I'd restore the second sentence. Seems helpful to me. Sorry if I mislead you into removing it the first time around.